src/billsdepositspanel.cpp

```diff
@@ -451,7 +449,10 @@ wxString mmBillsDepositsPanel::getItem(long item, long column)
     case COL_STATUS:
         return bill.STATUS;
     case COL_CATEGORY:
-        return bill.CATEGNAME;
+        if (bill.has_split())
+            return _("Split.......");
```

Comment:
I've suggest the same changes as here:
https://github.com/moneymanagerex/moneymanagerex/pull/476

~~~~~~~
-    CATEGNAME = Model_Category::full_name(r.CATEGID, r.SUBCATEGID);
+    if (!m_bill_splits.empty())
+    {
+        for (const auto& entry : m_bill_splits)
+            CATEGNAME += (CATEGNAME.empty() ? " * " : ", ")
+            + Model_Category::full_name(entry.CATEGID, entry.SUBCATEGID);
+    }
+    else
+        CATEGNAME = Model_Category::full_name(r.CATEGID, r.SUBCATEGID);
~~~~~~